server/containers/rpmrepo/config/sshd/sshd_config

```diff
@@ -1,6 +1,6 @@
 StrictModes no
 PasswordAuthentication no
-Subsystem sftp /usr/lib/openssh/sftp-server
+Subsystem sftp internal-sftp -u 0002
```

Comment:
sftp-server is a separate process, essentially an old version of the faster functionality built into sshd now. Since we were forcing the internal-sftp command anyway, doesn't make sense to set up sftp-server here.